src/internal/components/session/CommonSessionCard.tsx

```diff
@@ -10,16 +10,44 @@ interface ICommonSessionCardProps {
     cardName: string;
     faIcon: string;
     cardInfo?: string;
-    children?: (exportSession: SessionAction, cloneSession: SessionAction, saveSession?: SessionAction, deleteSession?: SessionAction) => React.ReactNode;
+    children?: (sessionAction: SessionActionChooser) => React.ReactNode;
 }
 
-type SessionAction = (event: React.MouseEvent<DropdownItemProps>, desc: IProvenanceGraphDataDescription, callback?: (value: React.SetStateAction<IProvenanceGraphDataDescription[]>) => void) => boolean | Promise<boolean>;
+
+/**
+ * Types of actions exposed by the CommonSessionCard component
+ */
+export const enum Action {
```

Comment:
```suggestion
export const enum EAction {
```

Prefix enums with an `E`, similar to interfaces that are prefixed with `I`.